pup-0006.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Commit Bit Assignment
+author: Ina Panova
+created: 15-08-2018
+status: Approved
+version: 1
+---
+
+## Summary
+
+For pulpcore and its related tools, there is no written process to describe giving the commit bit
+to a contributor. The existing committers want to openly document the process.
+
+
+## Motivation
+
+Comitter - is an individual who has write permissions to a
+[Pulp Orpanization](https://github.com/pulp/) repository.
+To have "a commit bit" means that one is permitted to commit those changes. 
+
+Historically the commit bit was given on day 0 when hired on the core team at Red Hat.  In Oct 2017
```

Comment:
Add: In practice, engineers did not act as commit bit owners from day 0 nor for all repositories in Pulp Organization.

Reply:
I guess when it comes to motivation, we'd better emphasize we want an open process instead